contracts/masset/platform-integrations/AaveIntegration.sol

```diff
@@ -115,6 +115,58 @@ contract AaveIntegration is InitializableAbstractIntegration {
         return _checkBalance(aToken);
     }
 
+    /**
+     * @dev Move aTokens from Aave v1 to v2
+     *      Migrates from Aave v1 to Aave v2
+     * @param _bAsset     Address of the bAsset
+     * @param _newAToken  Address of the newAToken
+     * @param _platformAddress  Address of the platform
+     */
+    //function migrate(address[] calldata _bAssets, address[] calldata _newATokens)
+    function migrate(address _bAsset, address _newAToken, address _platformAddress)
+        external
+        onlyGovernor
+    {
+        // TODO - handle array of _bAssets and _newATokens
+
+        // Get the existing Aave Platform Token for the bAsset
+        IAaveAToken aToken = _getATokenFor(_bAsset);
+
+        // Get the balance held on the contract
+        uint256 _aTokenBalance = _checkBalance(aToken);
+
+        // Redeem the underlying tokens from Aave v1
+        aToken.redeem(_aTokenBalance); // Handle revert
+
+        // Modify platform address if it has changed for v2
+        if (platformAddress != _platformAddress) {
+          _setPlatformAddress(_platformAddress);
```

Comment:
Probably unecessary to have a function for this, since its internal and only called once, here in this func.